modules/cpmso-installing.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cpmso-about.adoc
+
+:_content-type: PROCEDURE
+[id="cpmso-installing_{context}"]
+= Installing the Control Plane Machine Set Operator
+
+To use this Operator on Azure clusters and clusters on providers that do not support failure domains, you must install the Operator.
```

Comment:
The wording here confuses me, the operator is there and running, what's missing is the CR. Users need to create the CR to make the operator function, is this a docs thing?

Reply:
Nope, this is me missing a second place I should have updated this wording. Thanks!